client/blocks/reader-full-post/index.jsx

```diff
@@ -487,7 +490,11 @@ export default connect(
 
 		const { site_ID: siteId, is_external: isExternal } = post;
```

Comment:
What's the relationship between the `blogId` prop and `post.site_ID`? Is it always the same thing when the post was obtained with `getPostByKey( state, { feedId, blogId, postId } )`? Or can it happen that `feedId` prop is specified, `blogId` isn't and `post.site_ID` is the only place where the site ID is?

Reply:
if `blogId` exists then it should be identical to `post.site_ID` assuming `isExternal` is falsey.

> Or can it happen that feedId prop is specified, blogId isn't and post.site_ID is the only place where the site ID is?

Exactly! Since `FullPost` is rendered directly from the controller at `client/reader/full-post/controller`, the only way it has to determine the feedId/blogId is through the current url.  Depending on if the url is for a feed or a blog it will only pass through one or the other to FullPost.  By inspecting the corresponding `post` in state it can deduce whether or not it is a .com connected post